modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,18 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.4.2 - 2023-04-26
+
+=== Fixed
+
+* Selection was not correctly scrolled horizontally into view when using the `selection.scrollIntoView` API.
+* Tab navigation no longer incorrectly stops at menu buttons within toolbar groups.
+* The editor displayed a notification error when it failed to retrieve a blob image uri.
+* Setting an invalid unit in the `fontsizeinput` changed it to the default value instead of reverting it to the previous, and valid, value.
+* The `urlinput` dialog component would not open the type-ahead dropdown when the input value was reset to an empty string.
+* When dragging image elements and dropping the image in the editor the `dragend` event would sometimes not fire on Firefox.
+* Context toolbars displayed the incorrect status for the `advlist` plugin buttons.
+* The image would not be inserted when the `quickimage` button was used on Chrome.
```

Comment:
```suggestion
=== Fixed

* The editor displayed a notification error when it failed to retrieve a blob image uri.
* Tab navigation no longer incorrectly stops at menu buttons within toolbar groups.
* The `urlinput` dialog component would not open the type-ahead dropdown when the input value was reset to an empty string.
* Setting an invalid unit in the `fontsizeinput` changed it to the default value instead of reverting it to the previous, and valid, value.
* Selection was not correctly scrolled horizontally into view when using the `selection.scrollIntoView` API.
* Context toolbars displayed the incorrect status for the `advlist` plugin buttons.
* The image would not be inserted when the `quickimage` button was used on Chrome.
* When dragging image elements and dropping the image in the editor the `dragend` event would sometimes not fire on Firefox.
```
Entry order now matches that in TinyMCE’s changelog file, `/github/tinymce/modules/tinymce/CHANGELOG.md`.